### PR TITLE
FR Translation - new format on Swirle's translation branch 

### PR DESCRIPTION
**Pull request by HughMcZus**

### CODE REVIEW

**Comment by swirle13** [comment]
There are a few changes that are in my new commits that aren't in here.

- `common` was updated to remove some unused values https://github.com/swirle13/The-Witcher-Old-World-Tool/commit/eae76fe490647e28c051f4c019654e2d739f43bf#diff-f42562c98e440b987ac05f6dd6a75797b81b5d6a090db09faaee014f0478aff6L2-L26
- `setupHelper.base.playerSetup.markers` and `markersMages` were updated to reduce confusion on where to place the markers on the player cards: https://github.com/swirle13/The-Witcher-Old-World-Tool/commit/eae76fe490647e28c051f4c019654e2d739f43bf#diff-f42562c98e440b987ac05f6dd6a75797b81b5d6a090db09faaee014f0478aff6R235-R236
- `setupHelper.dagon` and `dagonMT` were updated to improve language and include Dagon Trail tokens https://github.com/swirle13/The-Witcher-Old-World-Tool/commit/eae76fe490647e28c051f4c019654e2d739f43bf#diff-f42562c98e440b987ac05f6dd6a75797b81b5d6a090db09faaee014f0478aff6R354-R355

I'm unsure if your changes include these differences in french, as I don't speak it, but from what it looks like, they don't appear to contain these changes.

**Comment by swirle13** [comment]
And this is my fault. I didn't double check that keys get updated when I run the script to add additional keys. I'll do this in my following commit(s)

**Comment by HughMcZus** [comment]
Done :) Thanks

File: src/locales/fr/translation.json [hunk 1]
```diff
@@ -436,153 +438,153 @@
...
+                "text": "Code source",
                 "link": "https://github.com/swirle13/The-Witcher-Old-World-Tool"
             }
         ],
-        "communities": "Communities",
+        "communities": "Communautés",
         "communitiesLinks": [
             {
-                "text": "$t(communityLinks.officialDiscord)",
+                "text": "Discord officiel",
```
Comment (swirle13) [inline src/locales/fr/translation.json:579]: can you update to `"text": "$t(communityLinks.officialDiscord)",`?

**Review by swirle13 — CHANGES_REQUESTED** [review]
Looks great so far, just a few of the recent changes need to be integrated and it should be good to go after that 👍🏼

**Review by swirle13 — APPROVED** [review]
Looks great, thank you so much!

**Pull request merged**